src/tests/profiler/native/multiple/multiple.cpp

```diff
@@ -0,0 +1,95 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "multiple.h"
+#include <thread>
+
+#define MAX_PROFILERS 3
+
+using std::thread;
+
+std::atomic<int> MultiplyLoaded::_exceptionThrownSeenCount(0);
+std::atomic<int> MultiplyLoaded::_detachCount(0);
+std::atomic<int> MultiplyLoaded::_failures(0);
+
+GUID MultiplyLoaded::GetClsid()
+{
+    // {BFA8EF13-E144-49B9-B95C-FC1C150C7651}
+    GUID clsid = { 0xBFA8EF13, 0xE144, 0x49B9, { 0xB9, 0x5C, 0xFC, 0x1C, 0x15, 0x0C, 0x76, 0x51 } };
+    return clsid;
+}
+
+HRESULT MultiplyLoaded::InitializeCommon(IUnknown* pICorProfilerInfoUnk)
```

Comment:
nit: Should this be multiple instead of multiply?

Reply:
I was using multiply as an adverb, not a verb 😄